scenarios/web/spring-boot/src/main/resources/static/css/site.css

```diff
@@ -0,0 +1,35 @@
+﻿body {

```

Comment:
everything in `resources/static` is a copy-paste of the static assets from the MVC template